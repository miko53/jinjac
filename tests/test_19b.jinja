-------------------------------
{% for x in range(0,10,1) %}
{% if x > 5 %}
Here x is > 5 --> x = {{x}}
{% else %}
And here x is < 5 --> x = {{x}}
{% endif %}
{% endfor %}
------------------------------
{% for y in range(0,10) %}
{% if users[y] %}
user exists ==> {{ users[y] }}
{% else %}
no users
{% endif %}
{% endfor %}
-- end of test --
