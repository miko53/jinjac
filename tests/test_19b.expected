-------------------------------


And here x is < 5 --> x = 0



And here x is < 5 --> x = 1



And here x is < 5 --> x = 2



And here x is < 5 --> x = 3



And here x is < 5 --> x = 4



And here x is < 5 --> x = 5



Here x is > 5 --> x = 6



Here x is > 5 --> x = 7



Here x is > 5 --> x = 8



Here x is > 5 --> x = 9


------------------------------


user exists ==> dana



user exists ==> bob



user exists ==> john



no users



no users



no users



no users



no users



no users



no users


-- end of test --
